proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_nl_NL.properties

```diff
@@ -0,0 +1,62 @@
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=Je bent al met deze server verbonden!
+velocity.error.already-connected-proxy=Je bent al met deze proxy verbonden!
+velocity.error.already-connecting=Je probeert al verbinding te maken met een server!
+velocity.error.cant-connect=Kan geen verbinding maken met {0}: {1}
+velocity.error.connecting-server-error=Kan jou niet verbinden met {0}. Probeer het later opnieuw.
+velocity.error.connected-server-error=Er is een probleem opgetreden bij jouw verbinding met {0}.
+velocity.error.internal-server-connection-error=Er is een interne serververbindingsfout opgetreden.
+velocity.error.logging-in-too-fast=Je logt te snel in, probeer het later opnieuw.
+velocity.error.online-mode-only=Je bent niet aangemeld bij jouw Minecraft-account. Als je wel aangemeld bent bij jouw Minecraft-account, probeer dan jouw Minecraft-client opnieuw op te starten.
+velocity.error.player-connection-error=Een interne fout trad op bij jouw verbinding.
+velocity.error.modern-forwarding-needs-new-client=Deze server is alleen compatibel met Minecraft 1.13 en hoger.
+velocity.error.modern-forwarding-failed=Uw server heeft geen forwarding request naar de proxy gestuurd. Zorg ervoor dat de server is geconfigureerd voor Velocity forwarding.
+velocity.error.moved-to-new-server=Je bent verwijderd van {0}: {1}
+velocity.error.no-available-servers=Er zijn geen beschikbare servers om jou mee te verbinden. Probeer het later opnieuw of neem contact op met een administrator.
+
+# Commands
+velocity.command.generic-error=Een fout trad op bij het uitvoeren van dit commando.
+velocity.command.command-does-not-exist=Dit commando bestaat niet.
+
+velocity.command.players-only=Alleen spelers kunnen dit commando uitvoeren.
+velocity.command.server-does-not-exist=De opgegeven server {0} bestaat niet.
+
+velocity.command.server-current-server=Je bent op dit moment verbonden met {0}.
+velocity.command.server-too-many=Er zijn te veel servers ingesteld. Gebruik tab complete om alle beschikbare servers te bekijken.
+velocity.command.server-available=Beschikbare servers:
+velocity.command.server-tooltip-player-online={0} spelers online
+velocity.command.server-tooltip-players-online={0} spelers online
+velocity.command.server-tooltip-current-server=Momenteel verbonden met deze server
+velocity.command.server-tooltip-offer-connect-server=Klik om te verbinden met deze server
+
+velocity.command.glist-player-singular=speler
+velocity.command.glist-player-plural=spelers
+velocity.command.glist-total-suffix= zijn momenteel verbonden met de proxy.
+velocity.command.glist-view-all=Om alle spelers op de servers te bekijken, gebruik /glist all.
+
+velocity.command.reload-success=Velocity configuratie succesvol herladen.
+velocity.command.reload-failure=Uw Velocity-configuratie kan niet opnieuw worden geladen. Kijk op de console voor meer details.
```

Comment:
`Je Velocity-configuratie` voor consistentie?

Reply:
`De Velocity-configuratie` kan ook.

Reply:
Misschien `De Velocity-configuratie` zodat het wat algemener is.